### PR TITLE
HPCC-27950 ES Translations Re-Add Missing

### PR DESCRIPTION
**Pull request by kunalaswani**
Translations re-added for Spanish.

Signed-off-by: Kunal Aswani <Kunal.Aswani@lexisnexisrisk.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [x] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

**Review by rpastrana — APPROVED** [review]
@kunalaswani seems complete. (lines 2-39)
Do we know how this happened (can we do something to prevent it in future?)
Will we ever remove messages? If not, we can make it a point of emphasis in code review.
Also, the last re-introduced message contains a bug -Advanzado should be Avanzado (prob needs its own Jira/PR)

**Review by rpastrana — COMMENTED** [review]
@kunalaswani was comparing line by line, and noticed some messages seem to contain expanded characters, I'll comment on each line, please confirm...

File: esp/src/src/nls/es/hpcc.ts [hunk 1]
```diff
@@ -1,4 +1,42 @@
...
+    AboutToLoseSessionInformation: "Está a punto de cerrar la sesión y perder toda la información de la sesión. ¿Desea continuar?",
+    Account: "Cuenta",
+    Action: "Acción",
+    Activate: "Activar",
+    Activated: "Activado",
+    ActivateQuery: "Activar Consulta",
+    ActivateQueryDeletePrevious: "Activar Consulta, ¿Borrar anterior?",
+    ActivateQuerySuspendPrevious: "Activar Consulta, suspender  anterior?",
+    Active: "Activo",
+    ActivePackageMap: "&lsquo;Package Map&rsquo; Activo",
```
Comment (rpastrana) [inline esp/src/src/nls/es/hpcc.ts:18]: This message seems to contain expanded characters?

File: esp/src/src/nls/es/hpcc.ts [hunk 2]
```diff
@@ -1,4 +1,42 @@
...
+    ActualSize: "Tamaño real",
+    Add: "Agregar",
+    AddAttributes: "Agregar atributos/valores a su método",
+    AddAttributes2: "Agregar atributos",
+    AddBinding: "Agregar conexión",
+    AddFile: "Añadir Archivo",
+    AddGroup: "Agregar Grupo",
+    AdditionalResources: "Recursos  Adicionales",
+    AddPart: "Agregar Parte",
+    AddProcessMap: "Agregar &lsquo;Package Map&rsquo;",
```
Comment (rpastrana) [inline esp/src/src/nls/es/hpcc.ts:33]: This message seems to contain expanded characters?

**Comment by kunalaswani** [comment]
@rpastrana escaped quotes should be fixed

**Comment by rpastrana** [comment]
@kunalaswani seems reasonable.

**Pull request merged**